### PR TITLE
HoTT release for 8.13

### PR DESCRIPTION
**Pull request by Alizter**
TODO:

- [ ] Create a tag in the HoTT library and point to the tag

### CODE REVIEW

**Comment by MSoegtropIMC** [comment]
@Alizter : thanks, looks good! I will copy it over to Coq platform and run a few CI and manual tests there on all supported platforms. Results should be available tomorrow.

**Comment by mattam82** [comment]
Shall we proceed here?

**Comment by MSoegtropIMC** [comment]
@mattam82 : as far as I understood @Alizter wanted to make a new tag. As soon as the tag is there, I can handle this quickly.

File: released/packages/coq-hott/coq-hott.8.13/opam [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+To use the HoTT library, the following flags must be passed to coqc:
+   -noinit -indices-matter
+To use the HoTT library in a project, add the following to _CoqProject:
+   -arg -noinit
+   -arg -indices-matter
+"""
+tags: [ "logpath:HoTT" ]
+url {
+  src: "https://github.com/HoTT/HoTT/archive/refs/heads/master.tar.gz"
+  checksum: "sha512=78a242db264fca6cc011783dadc1cc6da92c22e3e10d8405ceff72ea3c572f37a23cb3ab9c9c2bf417011a1f50c2534aaafb17adf9e6996463a2e2a95be2842c"
```
Comment (Alizter) [inline released/packages/coq-hott/coq-hott.8.13/opam:29]:
```suggestion
  src: "https://github.com/HoTT/HoTT/archive/refs/tags/CoqPlatform.8.13.1.tar.gz"
  checksum: "sha512=933595f61c4855eba57269963e2ad6e81c30e665cc17ada79b06b06ada4bb242881e1a5d9eafcdcad272a4d88d1e4216c3f418d4203f70301329b813da65d866"
```

**Review by MSoegtropIMC — APPROVED** [review]
LGTM

**Pull request merged**